### ISSUE TITLE
DB Update 1190 fails on postgresql 

### ISSUE THREAD

**Issue by ilu33** [redmatrix/hubzilla#773]
```sql
2017-05-14 03:07:43
alter table abook add abook_not_here int(11) not null default '0' ;
SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "("
LINE 1: alter table abook add abook_not_here int(11) not null defaul...
                                                ^
2017-05-14 03:07:43
create index abook_not_here on abook (abook_not_here);
SQLSTATE[42703]: Undefined column: 7 ERROR:  column "abook_not_here" does not exist
```
int(11) is not referenced in the postgresql docs. Sometimes where mysql uses int(11) postgresql uses serial but I'm not sure that would be right in this case.

**Comment by habeascodice** [comment]
int or smallint would be fine. The direct equivalent for future reference is numeric(11) - which BTW also works for mysql. However, an 11 digit numeric is a complete waste of space for something that will only be 0 or 1. I think smallint might also work for mysql. It should probably be smallint. Also, the default should not be text implicitly cast to numeric, that's not SQL compliant (it should be "default 0" not "default '0'").

**Comment by git-marijus** [comment]
This issue was fixed in ccf0274d13a9128b8c8471f727a98a827721d814
